.phpstorm.meta.php

```diff
@@ -63,6 +63,7 @@
 			'user_preferences'             => \Google\Web_Stories\User\Preferences::class,
 			'web_stories_block'            => \Google\Web_Stories\Block\Web_Stories_Block::class,
 			'video_captions'               => \Google\Web_Stories\Media\Video_Captions::class,
+			'video_muting'                 => \Google\Web_Stories\Media\Video_Muting::class,
```

Comment:
Should we prefix all media services with `media.`?

Reply:
Video caption like the line above doesn't have media at the start of it.

Reply:
That's why I am asking. Should we start doing this?

Just like the `rest.` prefix for REST controllers.

To keep the list tidy and organized.